### PR TITLE
Add FXIOS-4803 [v106] Python release notes script

### PR DESCRIPTION
**Pull request by lmarceau**
# [FXIOS-4803](https://mozilla-hub.atlassian.net/browse/FXIOS-4803) https://github.com/mozilla-mobile/firefox-ios/issues/11672
First step towards automating release notes when tagging a GitHub version. End goal will be that we could trigger this through a Github Action. Intermediary step is that we can trigger it locally.

### CODE REVIEW

File: GithubReleaseScript/github_release.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    def build_version(cls, version_string: str):
+        if "." in version_string:
+            split = version_string.split(".")
+            return cls(int(split[0]), int(split[1]))
+        else:
+            return cls(int(version_string), 0)
+
+
+# The Firefox for iOS App Store URL
+firefox_url = 'https://apps.apple.com/ca/app/firefox-private-safe-browser/id989804926'
```
Comment (isabelrios) [inline GithubReleaseScript/github_release.py:54]: Is it safe to commit this id? I guess if you have added it but just to double check :)

File: GithubReleaseScript/github_release.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    Whenever we fetch from the GitHub API for a username, we throttle as we can hit the rate limit rapidly.
+    This is also why we temporarily save the found usernames to avoid making too much API calls.
+    Returns:
+        A dictionary containing an array of commits for specific users.
+        Key being the username, value being the array of commits (of format #12345)
+    """
+    mention_commits = {}
+    saved_users = {}
+
+    g = Github(f"{config.username}", f"{config.token}")
```
Comment (isabelrios) [inline GithubReleaseScript/github_release.py:133]: We have talked about this offline, that would need to change so this works on a github action

**Review by isabelrios — APPROVED** [review]
LGTM, good first step to have the github action in place

**Comment by lmarceau** [comment]
I'll merge and open another ticket to bring this script closer to working with Github action

**Review by hgeet — COMMENTED** [review]
new update facebook

**Pull request merged**